prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,19 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+
+console.log(languages);
+
+const firstElement = languages[0];
+
+console.log(firstElement);
+
+const thirdElement = languages[2];
+
+console.log(thirdElement);
+
+const length = languages.length;
+
+console.log(length);
+
+const lastElement = languages[5];
```

Comment:
using the value works, but we want a dynamic solution. You want to use the length variable here as noted in the instructions and subtract one from it to target the final index

Reply:
Adjusted, thank you! 

![adjustment](https://user-images.githubusercontent.com/91649194/222919346-54b7ac69-703d-4a68-949d-076b35c7425a.PNG)